aten/src/ATen/native/vulkan/VulkanOpaqueTensorImpl.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <ATen/OpaqueTensorImpl.h>
+
+namespace at {
+// The only difference from OpaqueTensorImpl is faking strides(), stride(),
+// is_contiguous(). The main intention for this is to be able to run torchscript
+// model on Vulkan backend. Strides are not supported on Vulkan side, plan to
+// support them.
+template <typename OpaqueHandle>
+struct CAFFE2_API VulkanOpaqueTensorImpl
+    : public OpaqueTensorImpl<OpaqueHandle> {
+  VulkanOpaqueTensorImpl(
+      at::DispatchKeySet key_set,
+      const caffe2::TypeMeta& data_type,
+      c10::Device device,
+      OpaqueHandle opaque_handle,
+      c10::IntArrayRef sizes,
+      c10::IntArrayRef strides)
+      : OpaqueTensorImpl<OpaqueHandle>(
+            key_set,
+            data_type,
+            device,
+            opaque_handle,
+            sizes) {
+    strides_ = strides.vec();
```

Comment:
Consider initializing in the constructor's initializer list to avoid an unnecessary default initialization.